test/fail_compilation/no_object.d

```diff
@@ -0,0 +1,15 @@
+/*
+DFLAGS:
+REQUIRED_ARGS:
```

Comment:
Try `-v` here to debug win32

Reply:
I did.  See the result here: https://auto-tester.puremagic.com/show-run.ghtml?projectid=1&runid=3027610&isPull=true

> import    object	(..\..\druntime\import\object.d)

I don't know where that's coming from.

Reply:
> (edit: I just tried this for you)

You'll probably want to disable other platforms or the auto-tester will fail and will never get around to running the test in question.

Reply:
Oh sorry. Feel free to drop my commit. 
We should really print `D_FLAGS` with `-v` too ...

Reply:
> We should really print D_FLAGS with -v too ...

https://github.com/dlang/dmd/pull/7865 - this might help you.